MERL-1004: fixed missing edit button in the Faust toolbar when previewing a post (#1556)
wpengine/faustjs: .changeset/funny-starfishes-melt.md

---
"@faustwp/core": patch
---

> Bug where review routes missing the "Edit Post" toolbar node fixed

Reviewer: ```suggestion
Fixed bug where the "Edit Post" link was missing in the Faust Toolbar
```
